src/main/java/com/task/amazon/utils/ParseService.java

```diff
@@ -0,0 +1,5 @@
+package com.task.amazon.utils;
+
+public interface ParseService {
```

Comment:
@karsv do not push empty classes or interfaces in PR. Let's add them in the next PR as well as their implementations